### PR TITLE
Improvement scss to reduce duplicates part 1

### PR DESCRIPTION
**Pull request by apolokaklab**
**Part 1**
## Proposed Changes

- Improvements with use placeholder bulma for Reduces sizes .css

### CODE REVIEW

File: src/scss/components/_switch.scss [hunk 1]
```diff
@@ -5,11 +5,11 @@ $switch-padding: 0.2em !default;
 $switch-active-background-color: $primary !default;
 
 .switch {
+    @extend %unselectable;
```
Comment (afwn90cj93201nixr2e1re) [inline src/scss/components/_switch.scss]: It's doing same thing(not sure) as @input, so, any reasons for that?
изображение

Better to improve that stuff: https://github.com/jgthms/bulma/pull/2701

When we use styling(not prepared buefy dist) inside our app, buefy stuff, like @extend classname(not placeholder) creating too much unnecessary rules.

Comment (apolokaklab) [reply src/scss/components/_switch.scss]: @afwn90cj93201nixr2e1re what [this](https://github.com/jgthms/bulma/pull/2701#issuecomment-554651920)? you solution would not make sense

This only for duplicates and sizes file, please look `dist/*.css`, you can found many duplicates.

Comment (afwn90cj93201nixr2e1re) [reply src/scss/components/_switch.scss]: Read sass docs.

Comment (apolokaklab) [reply src/scss/components/_switch.scss]: Btw If you think so, why don't you support it?

Comment (afwn90cj93201nixr2e1re) [reply src/scss/components/_switch.scss]: Coz you r changing let b = 1, to let b = 1-0.

Can you show diffs between was and now?

Coz extending placeholder which have only iclude under, same as directly using input. Not sure about it, but without diffs its seems like same.

About @extend classname, buefy now has a bug. Its extending classname directly instead placeholder, thats why buefy taking user space in no prebuilt css using.

Comment (service-paradis) [reply src/scss/components/_switch.scss]: extend vs include does make a difference.
In this case, here is the difference between now and after @apolokaklab changes:

### before

```
...
.delete, .modal-close, .is-unselectable, .button, .file, .breadcrumb, .pagination-previous,
.pagination-next,
.pagination-link,
.pagination-ellipsis, .tabs, .carousel {
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
...
.b-checkbox.checkbox {
  outline: none;
  display: inline-flex;
  align-items: center;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
...
.b-radio.radio {
  outline: none;
  display: inline-flex;
  align-items: center;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
...
.switch {
  cursor: pointer;
  display: inline-flex;
  align-items: center;
  position: relative;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
```

### after

```
...
.delete, .modal-close, .is-unselectable, .button, .file, .breadcrumb, .pagination-previous,
.pagination-next,
.pagination-link,
.pagination-ellipsis, .tabs, .carousel, .b-checkbox.checkbox, .b-radio.radio, .switch {
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
...
.b-checkbox.checkbox {
  outline: none;
  display: inline-flex;
  align-items: center;
}
...
.b-radio.radio {
  outline: none;
  display: inline-flex;
  align-items: center;
}
...
.switch {
  cursor: pointer;
  display: inline-flex;
  align-items: center;
  position: relative;
}
```

Not a huge difference, but surely an improvement.

**Comment by afwn90cj93201nixr2e1re** [comment]
@jtommy merge it now.

**Pull request merged**